src/backend/aspen/workflows/nextstrain_run/builder_base.py

```diff
@@ -56,7 +56,10 @@ def update_build(self, config):
         build["subsampling_scheme"] = self.subsampling_scheme
 
         # Update the tree's title with build type & location.
-        title_template = "{tree_type} tree for samples collected in {location}"
+        if self.subsampling_scheme == "OVERVIEW":
+            title_template = "Contextualized tree for samples collected in {location} in the last 3 months"
+        else:
+            title_template = "{tree_type} tree for samples collected in {location}"
```

Comment:
this could go to builder_config.py, but having the concatenated {location} here is too convenient lol